libs/core/models/anime.model.ts

```diff
@@ -0,0 +1,67 @@
+import { BroadCastedTime } from './broadcasted-time.model';
+
+import { AnimeType } from './amime-type';
+import { AnimeStatus } from './anime-status';
+
+import { Immerable, OmitImmerable } from './immerable';
+
+/** Models of anime . */
+export class Anime extends Immerable {
+	/** Id of anime. */
+	public readonly id: number;
+
+	/** Date that anime was created. */
+	public readonly createdDate: Date;
+
+	/** Last modified time. */
+	public readonly modifiedDate: Date;
+
+	/** Title in English. */
+	public readonly englishTitle: string;
+
+	/** Title in Japanese. */
+	public readonly japaneseTitle: string;
+
+	/** Cover image of the anime. */
+	public readonly coverImageUrl: string;
+
+	/** Broadcast time frame from start to end of a anime. */
+	public readonly broadcasted: BroadCastedTime;
+
+	/** Type of the anime ex OVA. */
+	public readonly animeType: AnimeType;
+
+	/** Status of the anime. */
+	public readonly animeStatus: AnimeStatus;
+
+	/** Score of the anime from website. */
+	public readonly score: number | null;
+
+	/** Score from users. */
+	public readonly userScore: number | null;
+
+	/** Studio's id as array of string. */
+	public readonly studios: readonly string[];
+
+	/** Genres' id as array of string. */
+	public readonly genres: readonly string[];
```

Comment:
Let's use more descriptive names for these properties.
```suggestion
	/** Studio's id as array of string. */
	public readonly studioIds: readonly string[];

	/** Genres' id as array of string. */
	public readonly genreIds: readonly string[];
```